src/components/Button/Button.module.css

```diff
@@ -34,6 +39,10 @@
    &:disabled:hover {
     cursor: not-allowed;
   }
+
+  @media screen and (prefers-reduced-motion) {
+    transition: none;
+  }
```

Comment:
This isn't really important because the button is just changing color and it's really fast, but our current guideline is to add it whenever you have a transition.